ssn/integrated/sosa-oms.ttl

```diff
@@ -0,0 +1,476 @@
+  owl:inverseOf sosa:hosts ;
+  rdfs:isDefinedBy sosa: .
+
+
+## Procedures
+
+sosa:Procedure ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/Procedure> ;
+  rdfs:isDefinedBy sosa: .
+
+sosa:ObservingProcedure a rdfs:Class , owl:Class ;
+  ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/ObservingProcedure> ;
+  rdfs:subClassOf sosa:Procedure ;  
+  rdfs:label "Observing Procedure"@en ;
+  skos:definition "The description of steps performed in order to determine the value of an ObservableProperty by an Observer."@en ;
+  rdfs:comment "The description of steps performed in order to determine the value of an ObservableProperty by an Observer."@en ;
+  skos:example "A workflow, protocol, plan, algorithm, or computational method specifying how to make an observation; the description of the process utilized by an observer. This could be a chemical analysis method, a protocol for measuring an object, but could also be a checklist utilized by a human observer during a biodiversity campaign."@en ;
+  skos:note "The observing procedure cannot describe a sensor instance, but it can describe the sensor type."@en ;
+  rdfs:isDefinedBy sosa-oms: .
+
+sosa:ActuationProcedure a rdfs:Class , owl:Class ;
+  rdfs:subClassOf sosa:Procedure ;
+  rdfs:label "Actuation Procedure"@en ;
+  skos:definition "The description of steps performed by an Actuator to change the ActuableProperty."@en ;
+  rdfs:comment "The description of steps performed by an Actuator to change the ActuableProperty."@en ;
+  rdfs:isDefinedBy sosa-oms: .
+
+sosa:SamplingProcedure a rdfs:Class , owl:Class ;
+  ogc:implements <http://www.opengis.net/spec/om/3.0/req/sam-cpt/SamplingProcedure> ;
+  rdfs:subClassOf sosa:Procedure ;
+  rdfs:label "Sampling Procedure"@en ;
+  skos:definition "The description of steps performed by a Sampler in order to extract a Sample from its sampled Feature in the frame of a Sampling."@en ;
+  rdfs:comment "The description of steps performed by a Sampler in order to extract a Sample from its sampled Feature in the frame of a Sampling."@en ;
+  rdfs:isDefinedBy sosa-oms: .
+
+sosa:PreparationProcedure a rdfs:Class , owl:Class ;
+  ogc:implements <http://www.opengis.net/spec/om/3.0/req/sam-cpt/PreparationProcedure> ;
+  rdfs:subClassOf sosa:Procedure ;
+  rdfs:label "Preparation Procedure"@en ;
+  skos:definition "The description of preparation steps performed on a Sample."@en ;
+  rdfs:comment "The description of preparation steps performed on a Sample."@en ;
+  rdfs:isDefinedBy sosa-oms: .
+
+
+## ProcedureExecutors
+
+sosa:Sensor ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/Observer> ;
+#  rdfs:subClassOf sosa:Observer ;
+  rdfs:subClassOf sosa:System ;
+  rdfs:isDefinedBy sosa: .
+
+  sosa:observes ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/Observer/observableProperty-sem> ;
+    rdfs:subPropertyOf sosa:forProperty ;
+    owl:inverseOf sosa:isObservedBy ;
+#    schema:domainIncludes sosa:Observer ;
```

Comment:
Is there a reason this was removed from `sosa.ttl` but commented out in `sosa-oms.ttl`?

Reply:
That was waiting on the resolution of #29 and #43. 
Now removed.